source/npm/qsharp/ux/tsconfig.json

```diff
@@ -1,8 +1,11 @@
 {
   "compilerOptions": {
-    "module": "CommonJS",
+    "outDir": "../dist",
+    "declaration": true,
+    "sourceMap": true,
+    "module": "nodenext",
+    "moduleResolution": "nodenext",
     "target": "ES2020",
-    "noEmit": true,
```

Comment:
Do we build these just for the tests now? I see they build to the ./dist folder, so it means they will now be a new artifact of the shipping npm package, is that right? Previously they were just source, and didn't have their own build, as they were only consumed and included in the build of other components (such as jupyter widgets or VS Code)

Reply:
That's right.

I put them in `dist` because that just seemed to align the best with the current code (all the tests under `npm/qsharp/test` reference product code from `../dist`) . 

Shipping them does feel a bit unnecessary though. 

Perhaps I can exclude `dist/ux` from publishing in `package.json` ? Would that fit with your vision of how this is supposed to work?